notebook/static/bidi/bidi.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+define([
+    'bidi/numericshaping',
+], function (numericshaping){
+    "use strict";
+    
+    var shaperType="";
+    var textDir="";
+    
+    var _uiLang= function (){
+    	return navigator.language.toLowerCase();
+    };
+
+	var _loadLocale = function () {
+		if(_isMirroringEnabled()){
+			$("body").attr("dir","rtl");
+		}
+		requirejs(['components/moment/locale/'+_uiLang()], function (err){
```

Comment:
I _think_ we need to special case `en` and `en-us` since the files are not there and are the builtins from moment: https://github.com/moment/moment/issues/2081

Reply:
Looks like you already took care of it! 👍

Reply:
Oh yeah I should have noted that again here. Good to go now.